setup-demo.sh

```
@@ -0,0 +1,285 @@
+    echo -e "   • Check contexts: ${YELLOW}kubectl config get-contexts${NC}"
+    echo -e "   • Goat scenarios: ${YELLOW}https://madhuakula.com/kubernetes-goat/${NC}"
+    echo ""
+    echo -e "${GREEN}═══════════════════════════════════════════════════${NC}"
+}
+
+#######################################################################
+# Main Execution
+#######################################################################
+
+main() {
+    local start_time=$(date +%s)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)